sources/packages/api/src/services/institution-location/institution-location.service.ts

```diff
@@ -125,6 +119,11 @@ export class InstitutionLocationService extends RecordDataModelService<Instituti
       .select("location.id")
```

Comment:
I believe this method now returns only designated locations. Either method can be renamed or a comment can be added to mention that it returns only designated locations.

Reply:
I prefer renaming the method.